src/range-input/range-input.ts

```diff
@@ -77,7 +77,10 @@ export class NgAisRangeInput extends BaseWidget {
   public maxInputValue?: number | string = '';
 
   get step() {
-    const precision = parseNumberInput(this.precision) || 2;
+    const precision =
+      typeof this.precision !== 'undefined'
+        ? parseNumberInput(this.precision)
+        : 2;
```

Comment:
We can use the call property to define the default value no? Or is it provided as string? 

Anyway the default value should be `0`.